From b97a4dc4caf609975ba9e80336e63ebc9545a91b Mon Sep 17 00:00:00 2001
From: Erik Faye-Lund <erik.faye-lund@collabora.com>
Date: Thu, 6 Mar 2025 13:53:07 +0100
Subject: [PATCH] docs/features: add VK_EXT_hdr_metadata

This was missed when it was recently added.

Fixes: 4b663d561b8 ("vulkan/wsi: implement support for VK_EXT_hdr_metadata on Wayland")
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/33918>
---
 docs/features.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/docs/features.txt b/docs/features.txt
index 6e789b3ed29..53395bfee2a 100644
--- a/docs/features.txt
+++ b/docs/features.txt
@@ -612,6 +612,7 @@ Khronos extensions that are not part of any Vulkan version:
   VK_EXT_global_priority                                DONE (anv, hasvk, panvk, radv, tu)
   VK_EXT_global_priority_query                          DONE (anv, hasvk, panvk, radv, tu)
   VK_EXT_graphics_pipeline_library                      DONE (anv, lvp, nvk, panvk, radv, tu, vn)
+  VK_EXT_hdr_metadata                                   DONE (anv, lvp, nvk, panvk, radv, tu)
   VK_EXT_headless_surface                               DONE (anv, dzn, hasvk, lvp, nvk, panvk, pvr, radv, tu, v3dv, vn)
   VK_EXT_image_2d_view_of_3d                            DONE (anv, hasvk, lvp, nvk, radv, tu, vn)
   VK_EXT_image_compression_control                      DONE (radv)
-- 
2.49.0

